src/components/Generic/Button/Button.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Button from './Button';
+
```

Comment:
What functionality does this test check? Do we really need to test that button renders? Can we test more specific things?